api/tasks.js

```diff
@@ -19,7 +19,7 @@ export function createTasksApi({ api, config }) {
     }
 
     function getTaskDir(taskId) {
-        const escapedTaskId = taskId.replace(/\//g, '_');
+        const escapedTaskId = config.copyOriginToTaskDir ? taskId.replace(/\//g, '_') : '.';
```

Comment:
I would replace the `.` with `config.originDir`?  It might be working now because in most commands `cd $taskDir` is performed before running the command. But I wouldn't count on this too much :)

Reply:
probably I'm wrong :)